oxd-server/src/test/resources/testng.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="oxD Suite" parallel="false">
+<suite name="oxD Suite" parallel="true">
```

Comment:
Did you test parallel executions? I think we had problems during suite set up and tear down.